.azure/pipelines/ci.yml

```diff
@@ -277,6 +277,56 @@ stages:
       agentOs: Windows
       installNodeJs: false
       installJdk: false
+      steps:
+      - ${{ if ne(variables['System.TeamProject'], 'public') }}:
+        - task: PowerShell@2
+          displayName: Setup Private Feeds Credentials
+          inputs:
+            filePath: $(Build.SourcesDirectory)/eng/common/SetupNugetSources.ps1
+            arguments: -ConfigFile $(Build.SourcesDirectory)/NuGet.config -Password $Env:Token
+          env:
+            Token: $(dn-bot-dnceng-artifact-feeds-rw)
+            
+      - script: ./build.cmd
```

Comment:
No this isn't needed, the script is already defined below, this is a duplication. The two script steps and push vs packages should be removed. @wtgodbe are the private feeds credentials needed?

Reply:
No, main already does the right thing w.r.t SetupPrivateFeedCredentials. I think this PR should probably be a no-op, unless you think some of the win-arm64 stuff you did in 3.1 is needed to fix something in main

Reply:
I am waiting for the decision here before applying the changes I understand from the other ocmments.

Reply:
I don't think we want _any_ of the changes to this file - @JunTaoLuo do you agree?

Reply:
Agreed @wtgodbe. We don't want changes to .azure/pipelines/ci.yml in this PR.